buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -13,23 +13,25 @@ let Size = ./Size.dhall
 let Libp2p = ./Libp2pHelperBuild.dhall
 let DockerImage = ./DockerImage.dhall
 let DebianVersions = ../Constants/DebianVersions.dhall
+let DirtyWhen = [
```

Comment:
nit: Can you make this lower case (`dirtyWhen`), since it’s a normal value (a list) rather than a “module” . This is more of a style thing, but makes sense to me

Reply:
sure, fixed 👍